modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -67,7 +67,14 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   async _loadTileset(tilesetUrl) {
-    const {loader, loadOptions} = this.props;
+    const {loadOptions} = this.props;
+
+    // TODO: deprecate `loader` in v9.0
```

Comment:
Hey guys, just out of curiosity, is there any link to documentation or discussion on why loader will be deprecated in v9?

I was just looking around at how the loaders are implemented and surprised to see the comment.

Reply:
`loader` existed on `Tile3DLayer` prior to `loaders` being supported by all layers. We are not only going to use `loaders` moving forward for consistency.

Reply:
Fantastic! Thanks for the response and clarification!